openstack/cce/v3/addons/results.go

```diff
@@ -29,7 +25,7 @@ type MetaData struct {
 	Annotations map[string]string `json:"annotaions"`
 }
 
-// Specifications to create an addon
+// Spec to create an addon
```

Comment:
To match both `start with name` and `natural reading` rules, you can put it in `Spec - specification to create an add-on`